web/src/user_group_edit_members.ts

```diff
@@ -229,10 +263,17 @@ function add_new_members({pill_user_ids}) {
         });
     }
 
-    function invite_failure(xhr) {
+    function invite_failure(xhr?: JQuery.jqXHR): void {
         let message = "Failed to add user!";
-        if (xhr.responseJSON?.msg) {
-            message = xhr.responseJSON.msg;
+
+        const request_failure_response_schema = z
+            .object({
+                msg: z.string().min(1),
+            })
+            .safeParse(xhr?.responseJSON);
```

Comment:
The return value of `safeParse` is not a “schema”. I’d just name this `parsed`.

Reply:
Understood.